pkg/codegen/internal/test/testdata/provider-config-schema/python/pulumi_configstation/__init__.py

```diff
@@ -17,7 +17,7 @@
 
 _utilities.register(
     resource_modules="""
-null
+[]
```

Comment:
Python code in utilities.py was actually treating null/None and [] differently. This is correct.